### PR TITLE
remove Buffer constructor usage (#358)

### PR DESCRIPTION
**Pull request by gabrielschulhof**
* Replace occurrences of `new Buffer(...)` with `Buffer.alloc(...)` or
  `Buffer.from(...)`, as appropriate

### CODE REVIEW

**Comment by gabrielschulhof** [comment]
@mcollina IIUC I have removed the block.

File: examples/CAPSLOCKTYPER.JS [hunk 1]
```diff
@@ -11,7 +11,7 @@ function MyStream () {
 inherits(MyStream, Transform);
 
 MyStream.prototype._transform = function (chunk, outputFn, callback) {
-  outputFn(new Buffer(String(chunk).toUpperCase()));
+  outputFn(Buffer.alloc(String(chunk).toUpperCase()));
```
Comment (vweevers) [inline examples/CAPSLOCKTYPER.JS]: Shouldn't this be `.from`?

File: test/browser/test-stream-readable-event.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ function first(t) {
   };
 
   // This triggers a 'readable' event, which is lost.
-  r.push(new Buffer('blerg'));
+  r.push(Buffer.alloc('blerg'));
```
Comment (vweevers) [inline test/browser/test-stream-readable-event.js]: Same (also below)

File: examples/CAPSLOCKTYPER.JS (hunk 1 above)
Comment (mcollina) [reply examples/CAPSLOCKTYPER.JS]: ouch, tests ere passing!

Comment (mcollina) [reply examples/CAPSLOCKTYPER.JS]: @gabrielschulhof would you mind sending a follow-up PR?

**Comment by gabrielschulhof** [comment]
Can do.

**Pull request merged**